pkg/controller/container-runtime-config/helpers.go

```diff
@@ -8,11 +8,12 @@ import (
 	"fmt"
 	"reflect"
 	"strconv"
+	"strings"
 
 	"github.com/BurntSushi/toml"
 	"github.com/containers/image/docker/reference"
-	"github.com/containers/image/pkg/sysregistriesv2"
 	signature "github.com/containers/image/signature"
```

Comment:
Now that runtime-utils uses v5, please update this to image/v5/ as well, incl. `helpers_test.go`; that should make the v1 go away and decrease the code size.

Reply:
Fixed